src/NuGet.Core/NuGet.Build.Tasks/GetReferenceNearestTargetFrameworkTask.cs

```diff
@@ -242,19 +242,19 @@ private static NuGetFramework GetNuGetFramework(TargetFrameworkInformation targe
                     string.Empty :
                     targetFrameworkInformation._targetPlatformMoniker);
         }
-    }
-
-    internal class TargetFrameworkInformation
-    {
-        internal readonly string _targetFrameworkAlias;
-        internal readonly string _targetFrameworkMoniker;
-        internal readonly string _targetPlatformMoniker;
 
-        public TargetFrameworkInformation(string targetFrameworkAlias, string targetFrameworkMoniker, string targetPlatformMoniker)
+        internal class TargetFrameworkInformation
```

Comment:
Why do we have this change?

Reply:
I needed it because I made `InternalsVisibleTo` the other assembly and this class is declared in an outer scope rather than a subclass under `GetReferenceNearestTargetFrameworkTask`.  I don't think this class needs to be declared as a top-level class right?

Reply:
I see. 
I guess. Despite the fact that NuGet.Build.Tasks isn't always an entry assembly, we treat it as such, we don't use `PublicAPI` on it.